# One entry in a list of maps.

# Entire MapListEntry object
tele_presence/MapListEntry info

# Entire map object
nav_msgs/OccupancyGrid grid
